README.md

```diff
@@ -22,77 +22,80 @@ For more details, please, check the <a href="Sender.html">Sender</a>'s documenta
 ### Basic API usage
 
 ```javascript
-const { Sender } = require('@questdb/nodejs-client');
+const { Sender } = require("@questdb/nodejs-client")
 
 async function run() {
-    // create a sender using HTTP protocol
-    const sender = Sender.fromConfig('http::addr=localhost:9000');
-
-    // add rows to the buffer of the sender
-    await sender.table('prices').symbol('instrument', 'EURUSD')
-        .floatColumn('bid', 1.0195).floatColumn('ask', 1.0221)
-        .at(Date.now(), 'ms');
-    await sender.table('prices').symbol('instrument', 'GBPUSD')
-        .floatColumn('bid', 1.2076).floatColumn('ask', 1.2082)
-        .at(Date.now(), 'ms');
-
-    // flush the buffer of the sender, sending the data to QuestDB
-    // the buffer is cleared after the data is sent, and the sender is ready to accept new data
-    await sender.flush();
-
-    // add rows to the buffer again, and send it to the server
```

Comment:
This part of the example was supposed to demonstrate that you usually use the client by adding rows, then flush(), adding rows again, flush() again, and repeat until you call close() at the end.
Maybe it is not that important though, especially that auto-flushing is the default.